### PR TITLE
f-loyalty@v0.4.0 - adding in how it works and relevant tests

### PR DESCRIPTION
**Pull request by matthewhardern**
Adds in the How it works page into loyalty within the tabs component. I have added unit tests for covering the how it works page. Image links will be updated once the images have been uploaded into cloudinary.

## UI Review Checks

Screenshot 2021-08-24 at 10 11 03
Screenshot 2021-08-24 at 10 10 45
Screenshot 2021-08-24 at 10 10 32
Screenshot 2021-08-24 at 10 10 20

- [x] Unit tests have been [created|updated]

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### CODE REVIEW

File: packages/components/organisms/f-loyalty/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```
Comment (minrwhite) [inline packages/components/organisms/f-loyalty/CHANGELOG.md:5]: Double line break needed before first version listed:

```suggestion


```

File: packages/components/organisms/f-loyalty/src/components/HowItWorks.vue [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+            </a>
+        </div>
+    </div>
+</template>
+
+<script>
+import FCard from '@justeat/f-card';
+import MediaElement from '@justeat/f-media-element';
+
+const TEMP_IMG_PATH = 'https://d30v2pzvrfyzpo.cloudfront.net/b/hw/img/decoration/';
```
Comment (minrwhite) [inline packages/components/organisms/f-loyalty/src/components/HowItWorks.vue:117]: Note this will only work on average half the time due to relying on the pool that homeweb is active in - let's try and get cloudinary online soon as we can.

**Review by minrwhite — APPROVED** [review]

File: packages/components/organisms/f-loyalty/src/components/HowItWorks.vue (hunk 2 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/src/components/HowItWorks.vue:117]: I am arranging this today :), so should be solved soon

File: packages/components/organisms/f-loyalty/CHANGELOG.md (hunk 1 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/CHANGELOG.md:5]: yea will make the modification in the next PR

File: packages/components/organisms/f-loyalty/src/components/_tests/HowItWorks.test.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                i18n,
+                mocks: {
+                    $t: key => i18nMocker(key, i18n.locale)
+                }
+            });
+        });
+
+        it('bags should be calculated as expected', () => {
+            // Assert
+            expect(wrapper.vm.bags).toEqual([
```
Comment (DamianMullins) [inline packages/components/organisms/f-loyalty/src/components/_tests/HowItWorks.test.js:67]: Can do this in another PR, but this looks like a good candidate for a snapshot test? Same for the below.

**Review by DamianMullins — APPROVED** [review]

File: packages/components/organisms/f-loyalty/src/components/_tests/HowItWorks.test.js (hunk 3 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/src/components/_tests/HowItWorks.test.js:67]: good point will note it down thanks
